admin/common_auth/admin.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Permission
+from .models import MyUser
+
+from django.contrib.auth.forms import PasswordResetForm, UserCreationForm
+
+class PermissionAdmin(admin.ModelAdmin):
+    search_fields = ['name', 'codename']
+
+class CustomUserRegistrationForm(UserCreationForm):
```

Comment:
Form classes should be in forms.py. I know the custom admin example on the django site says to do this. It's a bad example :penguin: